.github/workflows/python_its-iqm.yml

```diff
@@ -0,0 +1,31 @@
+name: Python its-iqm
```

Comment:
Rename to its-interqueuemanager

Reply:
I renamed the top-level directory to _its-interqueuemanager_; I kept the filenames as _its-iqm_ (the entrypoint script, as well as the python module directory) as discussed.